### PR TITLE
Python demos fixes

### PR DESCRIPTION
**Pull request by mtsveta**
Incompatibilities with python3 are fixed and deprecated matplotlib attributes are replaced

### CODE REVIEW

**Comment by mtsveta** [comment]
@allanleal Looks like I don't have 'write access' to the repository, thus I cannot merge the PR.

**Comment by allanleal** [comment]
@mtsveta I sent you a member request a day or two ago. Please check your email. Once you accept it, it should give you such permissions. If not, let me know.

File: demos/python/demo-kineticpath-calcite-hcl.py [hunk 1]
```diff
@@ -15,36 +15,50 @@
...
 # along with this library. If not, see <http://www.gnu.org/licenses/>.
 
+# Step 1: Import the reaktoro Python package
 from reaktoro import *
 
+# Step 2: Define a chemical system with an aqueous and a mineral phase
 editor = ChemicalEditor()
-editor.addAqueousPhaseWithElementsOf("H2O HCl CaCO3")
+editor.addAqueousPhase("H2O HCl CaCO3")
+#editor.addAqueousPhaseWithElementsOf("H2O HCl CaCO3")
```
Comment (allanleal) [inline demos/python/demo-kineticpath-calcite-hcl.py]: Can we remove this commented line?

File: demos/python/demo-kineticpath-carbonates-co2.py [hunk 2]
```diff
@@ -15,16 +15,20 @@
...
 # along with this library. If not, see <http://www.gnu.org/licenses/>.
 
+# Step 1: Import the reaktoro Python package
 from reaktoro import *
 
+# Step 2: Add the phases of the chemical system
 editor = ChemicalEditor()
-editor.addAqueousPhaseWithElementsOf("H2O NaCl CaCO3 MgCO3")
+editor.addAqueousPhase("H2O NaCl CaCO3 MgCO3")
+#editor.addAqueousPhaseWithElementsOf("H2O NaCl CaCO3 MgCO3")
```
Comment (allanleal) [inline demos/python/demo-kineticpath-carbonates-co2.py]: We can also remove this commented line.

File: demos/python/demo-kineticpath-carbonates-co2.py [hunk 3]
```diff
@@ -90,7 +104,9 @@
...
 plot3.ylabel("Mass [g]")
 
-path.solve(state0, 0, 25, "hours")
-
-print state0
+# Step 11: Solve the kinetic paths
+t0, t1 = 0.0, 25.0
+path.solve(state0, t0, t1, "hours")
+# Print the result state
+print(state0)
```
Comment (allanleal) [inline demos/python/demo-kineticpath-carbonates-co2.py]: We should use:

`state0.output('demo-kineticpath-carbonates-co2-after-kinetics')`

File: demos/python/demo-reactive-transport-calcite-dolomite.py [hunk 4]
```diff
@@ -106,7 +108,7 @@ def simulate():
     b = zeros((ncells, nelems))
 
     # The concentrations of each element in each mesh cell (in the previous time step)
-    bprev = zeros((ncells, nelems))
+    # bprev = zeros((ncells, nelems))
```
[the remark this answers is not in the document]
Comment (mtsveta) [reply demos/python/demo-reactive-transport-calcite-dolomite.py]: No, we don't! But I just wanted to make sure that it wasn't meant to do something earlier.

**Pull request merged**